### PR TITLE
add getOnWakeUp

### PR DESCRIPTION
**Pull request by caseda**
added getOnWakeUp to capabilities that are not send automatically, or send frequently
this should solve most issues people have with `null` value

removed the "todo" remove code, since it is indeed not needed.

changed the GET/SET parsers to cleaner way

### CODE REVIEW

File: drivers/FGMS-001-PLUS/driver.js [hunk 1]
```diff
@@ -88,14 +82,12 @@ module.exports = new ZwaveDriver(path.basename(__dirname), {
...
-					'Sensor Type': 'Luminance (version 1)',
-					'Properties1': {
-						'Scale': 0
-					}
-				};
-			},
+			'command_get_parser': () => ({
+				'Sensor Type': 'Luminance (version 1)',
+				'Properties1': {
+					'Scale': 1,
```
Comment (RobinBol) [inline drivers/FGMS-001-PLUS/driver.js]: Why is the scale changed?

File: drivers/FGMS-001/driver.js [hunk 2]
```diff
@@ -46,14 +42,12 @@ module.exports = new ZwaveDriver( path.basename(__dirname), {
...
-					'Sensor Type': 'Luminance (version 1)',
-					'Properties1': {
-						'Scale': 0
-					}
-				};
-			},
+			'command_get_parser': () => ({
+				'Sensor Type': 'Luminance (version 1)',
+				'Properties1': {
+					'Scale': 1,
```
Comment (RobinBol) [inline drivers/FGMS-001/driver.js]: Again, scale change?

**Comment by caseda** [comment]
Look at the documentation :wink: not that it really matters since the get command itself is not in the code.

**Comment by RobinBol** [comment]
Well, it does matter, it is not the get parser is never used or something. But ok, if you found that in the docs it's fine, just asking to make sure it didn't happen by mistake.

**Pull request merged**